shpc/main/modules/templates/singularity.lua

```diff
@@ -46,6 +46,7 @@ local MODULEPATH="{{ module_dir }}"
 setenv("SINGULARITY_SHELL", "{{ singularity_shell }}")
 setenv ("SINGULARITY_OPTS", "")
 setenv ("SINGULARITY_COMMAND_OPTS", "")
+setenv ("SINGULARITY_COMMAND_ARGS", "")
```

Comment:
Ah this is a good catch, and I think it's a typo on my part to have `SINGULARITY_COMMAND_ARGS`
```suggestion
```